wallet/wallet.c

```diff
@@ -1291,6 +1291,22 @@ static struct channel *wallet_stmt2channel(struct wallet *w, struct db_stmt *stm
 		scid = NULL;
 	}
 
+	if (!db_col_is_null(stmt, "alias_local")) {
+		alias[LOCAL] = tal(tmpctx, struct short_channel_id);
+		if (!db_col_short_channel_id_str(stmt, "alias_local", alias[LOCAL]))
```

Comment:
Don't use db_col_short_channel_id_str in new code; encoding scids as strings in the db was a mistake.

We could create a new one, or (ab)use `db_col_short_channel_id_arr` which uses the wire encoding in a blob.